tests/test_c_interpolation_tables.py

```diff
@@ -168,8 +169,10 @@ def test_Massfunc_conditional_tables(name):
         up.INTEGRATION_METHOD_HALOS,
     )
 
+    max_in_d = N_cmfi_cell[:, :1]
+    max_in_d[:, 0] = 1.0  # fix delta=-1
     N_cmfi_cell = (
-        N_cmfi_cell / N_cmfi_cell[:, :1]
+        N_cmfi_cell / max_in_d
```

Comment:
something seems wrong here -- `max_in_d` is defined as all ones, then you're dividing by it, which does nothing?

Reply:
You're right, I reversed the indices. These are meant to be P(>M) (and so we divide by the first column at M = M_min) but with delta=-1 all entries are zero